src/MessagingBubble/MessagingBubble.tsx

```diff
@@ -6,12 +6,15 @@ import { matchDecorator, componentDecorator } from "./linkifyUtils";
 import "./MessagingBubble.less";
 
 const cssClasses = {
+  FLEXBOX: "flexbox",
   MESSAGE_BASE: "MessagingBubble--Message",
   MESSAGE_OWN: "MessagingBubble--Message--Own",
   MESSAGE_OTHER: "MessagingBubble--Message--Other",
   MESSAGE_REPLY_OWN: "MessagingBubble--Message--Reply--Own",
   MESSAGE_REPLY_OTHER: "MessagingBubble--Message--Reply--Other",
   MESSAGE_REPLY_PARENT: "MessagingBubble--Message--Reply--Parent",
+  MESSAGE_ATTACHMENT_OWN: "MessagingBubble--Message--Attachment--Own",
+  MESSAGE_ATTACHMENT_OTHER: "MessagingBubble--Message--Attachment--Other",
```

Comment:
I wonder if it would be clearer to say `Sent` and `Received` instead of `Own` and `Other`? just a thought, up to you